core/src/main/java/io/lionweb/lioncore/java/serialization/AbstractSerialization.java

```diff
@@ -220,7 +227,7 @@ public SerializedChunk serializeNodesToSerializationBlock(
   }
 
   private SerializedClassifierInstance serializeNode(
-      @Nonnull ClassifierInstance<?> classifierInstance) {
+      @Nonnull ClassifierInstance<?> classifierInstance, SerializationStatus serializationStatus) {
```

Comment:
If we hand the `serializationStatus` to every method, isn't it part of the serializer's state? Then, we can just make it a field.

Reply:
I think it is similar to deserializationStatus: we keep it while serving one request and we pass it to a few internal methods but when the request is satisfied the instance of serializationStatus is released, which means also the caches within serializationStatus are released